docs/config.md

```diff
@@ -108,17 +108,22 @@ Callables or other Python objects have to be passed in `setup.py` (via the `use_
 :   used as the primary source for the version number
     in which case it will be an unparsed string
 
-    !!! warning "it is strongly recommended to use use distribution name specific pretend versions"
+    !!! warning ""
 
+        it is strongly recommended to use distribution-specific pretend versions
+        (see below).
 
 `SETUPTOOLS_SCM_PRETEND_VERSION_FOR_${NORMALIZED_DIST_NAME}`
-:   used as the primary source for the version number
-    in which case it will be an unparsed string
+:   used as the primary source for the version number,
+    in which case it will be an unparsed string.
+    Specifying distribution-specific pretend versions will
```

Comment:
Can I say something like "distribution-specific (aka package-specific)"

Reply:
I'm wondering if it perhaps should change to python package name to disambiguate

Reply:
I finally got around to this today, and concluded:

1. The term "distribution" is all over the Python docs (e.g. [`importlib.metadata`](https://docs.python.org/3/library/importlib.metadata.html), [legacy install docs](https://docs.python.org/3.10/install/index.html), [`setuptools` docs](https://setuptools.pypa.io/en/latest/search.html?q=distribution&check_keywords=yes&area=default))
2. "dist" is also all over the place in the source code here

So I think it's fine to stick with the terminology "distribution" here after all. I am going to skip on opening a PR to change the wording from "dist" to "pkg"